scheduler/src/cook/mesos/api.clj

```diff
@@ -891,6 +890,10 @@
               progress-regex-string (assoc :progress-regex-string progress-regex-string)
               pool (assoc :pool (:pool/name pool))))))
 
+(defn fetch-job-map
+  [db framework-id job-uuid]
+  (fetch-job-map-from-entity db framework-id (d/entity db [:job/uuid job-uuid])))
```

Comment:
Does anything still use `fetch-job-map` after this change?

Reply:
Over a dozen uses.

Reply:
What are they? Should they switch over to using the new function?

Reply:
render-jobs-for-response-deprecated -- /rawscheduler handler
render-instances-for-response 
11 times as a convenience function in unit tests.

Reply:
Thats a scope question..

If we do that, then effectively we're effectively rescoping this PR to do all of #617. Do we want to do that?

Reply:
I would vote to either do that in this PR or do it immediately after this in a follow-up PR. Basically, I wouldn't want the two functions to stay around for very long.